documentation/modules/upgrading/con-downgrade-target-version.adoc

```diff
@@ -4,15 +4,40 @@
 
 [id='com-target-downgrade-version-{context}']
 
-= Target downgrade version
+= Kafka version compatibility for downgrades
 
-How the Cluster Operator handles a downgrade operation depends on the `log.message.format.version`.
+Kafka downgrades are dependent on compatible current and target xref:ref-kafka-versions-{context}[Kafka versions],
+and the state at which messages have been logged.
 
-* If the target downgrade version of Kafka has the same `log.message.format.version` as the current version, the Cluster Operator downgrades by performing a single rolling restart of the brokers.
-* If the target downgrade version of Kafka has a different `log.message.format.version`, downgrading is only possible if the running cluster has _always_ had `log.message.format.version` set to the version used by the downgraded version.
-+
+You cannot revert to the previous Kafka version if the `inter.broker.protocol.version` differs, or messages have been added
+to message logs that use a newer `log.message.format.version`.
+
+The `inter.broker.protocol.version` determines the schema used in the messages written to `__consumer_offsets`.
+If you downgrade to a version of Kafka that uses an earlier `inter.broker.protocol.version`, the broker will not understand the messages.
```

Comment:
While that's true, it doesn't inform the reader about what the consequences of that will be. We should be clear that, in general, it  basically results in a "broker cluster". The rest of this concept then goes into detailed talk about the log message format version which could easily mislead people into thinking this sentence is not really a problem. I think we should probably omit the following sentences because they're mostly irrelevant due to a broker cluster generally being a "game over" situation.

Reply:
We're leaving this section out for now. I'm not sure how much we will want to retain when this situation is rectified?

Reply:
I've updated this sentence to:

> Therefore, if you downgrade to a version of Kafka that uses an earlier `inter.broker.protocol.version`, the cluster will not work.